index.html

```diff
@@ -41,8 +41,54 @@
         .iti.iti--allow-dropdown{
           margin-left: 250px;
         }
-       
+        .container {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        flex-wrap: wrap;
+        width: 100%;
+        margin: 0;
+        padding: 0;
+    }
+
+    .content {
+        text-align: left;
+        flex: 1;
+        padding: 20px;
+    }
+
+    .sec-video {
+        flex: 1;
+        width: 20px;
+        height: auto; /* Maintain aspect ratio */
+    }
+
+    .btn-get-started {
+        display: inline-block;
+        background-color: #55a5ea;
+        color: #fff;
+        padding: 10px 20px;
+        text-decoration: none;
+        border-radius: 5px;
+        margin-top: 20px;
+    }
+
+    @media screen and (max-width: 768px) {

```

Comment:
I think a standard mobile screen max-width could be 530 px. Can you add this media screen too?